http/headers/feature-policy.json

```diff
@@ -294,6 +294,55 @@
             }
           }
         },
+        "battery": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/battery",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "notes": "Will be implemented, see <a href='https://crbug.com/1007264'>bug 1007264</a>."
+              },
+              "chrome_android": {
+                "version_added": false
```

Comment:
The bug note should be here as well.

Reply:
Is there a plan to ship `battery` for Android WebView? If so, I should insert that note too.

Reply:
Marked resolved because I added the note to `chrome_android`.